.github/workflows/test.action.yml

```diff
@@ -27,3 +27,8 @@ jobs:
       - run: yarn install
       - run: yarn build
       - run: yarn test
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        with:
+          build: yarn build
+          start: yarn start
```

Comment:
You don't have a start script in `package.json` btw. Also you don't have a script for starting cypress at all.